### ISSUE TITLE
After editing text in the block's editor the changes does not apply

### ISSUE THREAD

**Issue by bd0bd** [backdrop/backdrop-issues#1244]
After editing text in the Masthead and Button block's editor the changes does not apply after saving the block. So I was forced to delete this block and then create a new one.

![block](https://cloud.githubusercontent.com/assets/13515302/10058886/7428de4a-6250-11e5-8aa2-6b7cd71dc52a.gif)

**Comment by quicksketch** [comment]
Thanks again @bd0bd. You're quite right. It's odd after we spent time fixing the editor not being able to insert images within blocks that we wouldn't have caught this one. :(

**Comment by quicksketch** [comment]
PR filed at https://github.com/backdrop/backdrop/pull/1086 that fixes this.

We didn't have a .detach() method for editors, which is odd because we had already made a very specific detachEditor method that just needed to be called. Adding this detach method makes it so that the editor saves back into the textarea before the dialog form is submitted, and makes the value save properly.

**Comment by quicksketch** [comment]
This fix will be in 1.2.2. Thanks again @bd0bd!
